lte/gateway/configs/pipelined.yml

```diff
@@ -162,8 +162,13 @@ ovs_gtp_stats_polling_interval: 180
 he_proxy_eth_mac: 'e6:8f:a2:80:80:80'
 he_enabled: true
 
+# Paging timeout value
+paging_timeout: 30
+
 # Make as a True when gnb ip will support for uplink 
 ovs_multi_tunnel: False
+# Paging timeout value
+paging_timeout: 30
```

Comment:
i see this twice. This may mean this PR was not merged properly and other issues may have been introduced

Reply:
Will close the PR. Meant only to provide integrated UPF code for SMF to test (for the demo). As the UPF patches are individually landed this PR is not needed.